patches/subst.c

```diff
@@ -1,32 +0,0 @@
-diff --git a/subst.c b/subst.c
```

Comment:
I realized I forgot to add the extension .patch to this one.  I'm assuming that's why you deleted it? It fixes https://github.com/ZOSOpenTools/bashport/issues/5